lib/unicode_plot.rb

```diff
@@ -1,25 +1,30 @@
 require 'stringio'
 
-require 'unicode_plot/version'
+require_relative './unicode_plot/version'
```

Comment:
converted to relative requires because it makes development easier for me, but can revert if you prefer normal require.